src/ParameterPostProcessor.php

```diff
@@ -28,14 +28,19 @@ public function __construct(array $parameters)
         $this->parameters = $parameters;
     }
 
+    /**
+     * @return (array|mixed)[]
+     *
+     * @psalm-return array{parameters: array}
```

Comment:
This is not correct. The return value of this method returns more than just the parameters.
Please use the suggested annotation instead.
```suggestion
     * @psalm-return array<string,mixed>
```